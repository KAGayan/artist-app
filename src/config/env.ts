export const env = {
  API_KEY: process.env.REACT_APP_API_KEY,
  BASE_URL: process.env.REACT_APP_API_BASE_URL,
  ARTIST: process.env.REACT_APP_ARTIST,
};
