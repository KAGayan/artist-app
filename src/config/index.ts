export * from './env';
export * from './makeClasses';
