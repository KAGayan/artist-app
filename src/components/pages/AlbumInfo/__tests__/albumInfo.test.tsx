import '@testing-library/jest-dom';
import { render } from 'config';
import { AlbumInfo } from '..';

test('renders AlbumInfo', () => {
  window.scrollTo = jest.fn();

  const { asFragment } = render(<AlbumInfo />);

  expect(asFragment()).toMatchSnapshot();
});
