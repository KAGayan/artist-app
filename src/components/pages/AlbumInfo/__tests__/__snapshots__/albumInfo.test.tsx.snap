// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AlbumInfo 1`] = `
<DocumentFragment>
  <div
    class="css-urx51g-MuiStack-root"
  >
    <header
      class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionFixed mui-fixed css-11b3ww9-MuiPaper-root-MuiAppBar-root"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthXl css-19r6kue-MuiContainer-root"
      >
        <div
          class="MuiToolbar-root MuiToolbar-regular css-1cndkqf-MuiToolbar-root"
        >
          <div
            class="css-1o5q8ri-MuiStack-root"
          >
            <button
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
              tabindex="0"
              type="button"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1d159sf-MuiSvgIcon-root"
                data-testid="ArrowBackIosNewOutlinedIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M17.77 3.77 16 2 6 12l10 10 1.77-1.77L9.54 12z"
                />
              </svg>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
          <div
            class="css-eyffu-MuiStack-root"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
            >
              Eminem
            </p>
            <div
              class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-1qxe5qy-MuiAvatar-root"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiAvatar-fallback css-10mi8st-MuiSvgIcon-root-MuiAvatar-fallback"
                data-testid="PersonIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
                />
              </svg>
            </div>
          </div>
        </div>
      </div>
    </header>
    <div
      class="css-1n75z56-MuiStack-root"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthXl css-19r6kue-MuiContainer-root"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-12n0uxm-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
            >
              <div
                class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-1ri6ub7-MuiPaper-root-MuiCard-root"
              >
                <span
                  class="MuiSkeleton-root MuiSkeleton-rectangular MuiSkeleton-wave css-1slz293-MuiSkeleton-root"
                  style="width: 250px; height: 250px;"
                />
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item css-r4jj1p-MuiGrid-root"
            >
              <span
                class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-wave css-1dk1h2u-MuiSkeleton-root"
                style="height: 40px;"
              />
              <span
                class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-wave css-1dk1h2u-MuiSkeleton-root"
                style="width: 100px; height: 20px;"
              />
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
          >
            <span
              class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-wave css-1dk1h2u-MuiSkeleton-root"
              style="width: 200px; height: 60px;"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
